build/Dockerfile.windows

```diff
@@ -0,0 +1,65 @@
+# escape=`
```

Comment:
This PR no longer belongs in this repo.  Instead, it belongs here.

https://github.com/FoundationDB/fdb-build-support/tree/main/docker/windows

(Sorry for the churn.)

Reply:
This PR was created before the move and is outdated. The fdb-build-support repo already contains the same files. @moulmahdi is working on improving them to be able to run a CI build (build + tests) on top the the currently existing release build (installers, ...)